src/components/spreadsheet/utils/config-tables.js

```diff
@@ -179,33 +183,53 @@ export const TABLES_DEFINITIONS = {
                 undefined,
                 excludeFromGlobalFilter
             ),
-            generateEditableNumericColumnDefinition(
-                'IpMin',
-                'ipMin',
-                1,
-                'equipment.setIpMin({})\n',
-                true,
-                undefined,
-                'ipMax',
-                excludeFromGlobalFilter
-            ),
-            generateEditableNumericColumnDefinition(
-                'IpMax',
-                'ipMax',
-                1,
-                'equipment.setIpMax({})\n',
-                false,
-                'ipMin',
-                undefined,
-                excludeFromGlobalFilter,
-                {
+            {
+                id: 'IpMin',
```

Comment:
Why we not use functions ?

Reply:
generateEditableNumericColumnDefinition uses cellEditorParams, but in this case we need valueGetter and valueSetter